stdlib/bytes.mli

```diff
@@ -463,6 +463,106 @@ val of_seq : char Seq.t -> t
 (** Create a string from the generator
     @since 4.07 *)
 
+(** {6 Binary storage of integers} *)
+
+val get_uint16_ne : bytes -> int -> int
+(** [get_uint16_ne s n] reads a 16-bit unsigned integer from the byte
```

Comment:
I think this should be:
```
[get_uint16_ne b i] is [b]'s native endian unsigned 16-bit integer starting at byte index [i]. @since NEXT_RELEASE
```
so the pattern is: 
```
[get_SSintXX_EE b i] is [b]'s EE SS XX-bit integer starting at byte index [i]. @since NEXT_RELEASE
```